MediaAdminBundle/DataFixtures/MongoDB/LoadGroupFunctionalData.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace OpenOrchestra\MediaAdminBundle\DataFixtures\MongoDB;
+
+use Doctrine\Common\Persistence\ObjectManager;
+use OpenOrchestra\MediaAdminBundle\NavigationPanel\Strategies\TreeFolderPanelStrategy;
+use OpenOrchestra\ModelInterface\DataFixtures\OrchestraFunctionalFixturesInterface;
+
+/**
+ * Class LoadGroupFunctionalData
+ */
+class LoadGroupFunctionalData extends AbstractLoadGroupData implements OrchestraFunctionalFixturesInterface
+{
+    /**
+     * @param ObjectManager $manager
+     */
+    public function load(ObjectManager $manager)
+    {
+        $groupFoldersCreate = $this->generateGroup(
+            'Media folders create',
+            'Media folders create',
+            'Groupe de dossiers de la médiathèque ayant les droits access et create',
```

Comment:
i don't think so, this is a description it must be explicit or users may be confused